client/containers/Pub/PubDocument/PubMouseEvents/NotePopover.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import Popper from 'popper.js';
+import { Card } from '@blueprintjs/core';
+
+require('./notePopover.scss');
+
+const propTypes = {
+	elem: PropTypes.object.isRequired,
+	mainContentRef: PropTypes.object.isRequired,
+	timeouts: PropTypes.object.isRequired,
+	mouseLeave: PropTypes.func.isRequired,
```

Comment:
Since it's a callback, this prop should be named `onMouseLeave` for consistency's sake.

Reply:
Handled in d6864b3.